content/docs/2.4/scalers/azure-storage-blob.md

```diff
@@ -30,6 +32,9 @@ triggers:
 - `connectionFromEnv` - Name of the environment variable your deployment uses to get the connection string.
 - `blobPrefix` - Prefix for the Blob. Use this to specifiy sub path for the blobs if required. (default: `""`)
 - `blobDelimiter` - Delimiter for identifying the blob prefix. (default: `/`)
+- `cloud` - Name of the cloud environment that the blob belongs to. Must be a known Azure cloud environment, or `Private` for Azure Stack Hub or Air Gapped clouds. (default: `AzurePublicCloud`)
```

Comment:
We should list all allowed values here to help end-users

Reply:
Agree - done. The main reason I didn't add them initially was because I was worried it will never be updated when new cloud environments are added.